hw/ip/otbn/dv/otbnsim/test/simple/insns/add.exp

```diff
@@ -2,11 +2,11 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-x10 = 1
-x11 = 0xffffffff
-x2 = 2
-x3 = 3
-x4 = 2
-x5 = 0xfffffffe
+# x10 = 1
+# x11 = 0xffffffff
+# x2 = 2
+# x3 = 3
+# x4 = 2
+# x5 = 0xfffffffe
```

Comment:
We're effectively disabling all these tests. Do we need to do the secure wipe in the ISS at all? As suggested in another comment we could check the register writes OTBN is emitting are doing something sensible and have the ISS finish at end of the execute (either ecall or error), rather than end of secure wipe so these expected values still work. @rswarbrick what's your view?

Reply:
I think we should do what I said in the phone call on Tuesday: add two points at which we dump registers. For these ISS tests, we'd just check against the first point. For RTL testing, we'd probably check both. @vrozic, you said in the call that you understood and were happy to implement this. Is that still the case?

If not, I guess we should just try to get this PR merged ASAP. Rather than commenting everything out here, maybe it's better just to disable these tests and add a massive TODO comment. And file an issue, of course.

Reply:
I thought we said we'd dump to dmem and rely on the dmem compare for the smoke test at least?

Do these tests ever get run against RTL I thought they were ISS only?

Reply:
Yep, you're right: they're ISS only. I was thinking that if we put two hooks in then we can use them for the smoke test without having to mess around porting it to doing a load of `bn.sid` operations.